EnglishFilter/sections/general_extensions.txt

```diff
@@ -242,6 +242,8 @@ artipedia.id#%#//scriptlet("prevent-window-open")
 artipedia.id#%#//scriptlet('remove-class', 'get-link', 'a[class="btn btn-success btn-lg get-link"][target="_blank"]')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/74297
 elil.cc#%#//scriptlet('remove-attr', 'onmouseover|onclick|onmouseout', '.save-btn.pull-right')
+! https://github.com/AdguardTeam/AdguardFilters/issues/74255
+uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe-generate")
```

Comment:
This rule should be in `tweaks` section.
`uniqueten.net,ultraten.net#%#//scriptlet("adjust-setInterval", "wpsafe", "", "0.02")`

Reply:
Done.